docs/source/releasenotes/3.4.rst

```diff
@@ -0,0 +1,26 @@
+.. _Version34ReleaseNotes:
+
+3.4 Release Notes
```

Comment:
Please order numerically

Reply:
for sure, reordered!
<img width="895" alt="Screen Shot 2020-08-13 at 9 34 17 AM" src="https://user-images.githubusercontent.com/7649279/90161709-64fe6980-dd48-11ea-8463-db5a26b0a13e.png">